composites/Plugin/SnippetEditor/components/ReplacementVariableEditor.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { EditorState, convertToRaw, convertFromRaw } from "draft-js";
+import Editor from "draft-js-plugins-editor";
+import createMentionPlugin, { defaultSuggestionsFilter } from "draft-js-mention-plugin";
+import { serializeEditor, unserializeEditor } from "../serialization";
+import flow from "lodash/flow";
+import PropTypes from "prop-types";
+import { replacementVariablesShape } from "../constants";
+
+/**
+ * Creates a DraftJS editor state from a string.
+ *
+ * @param {string} content The content to turn into editor state.
+ *
+ * @returns {EditorState} The editor state.
+ */
+const createEditorState = flow( [
```

Comment:
`flow` is a magical thing, I cannot really guess what is happening here. Is there a way to make this more usable/readable?

Reply:
After you know what `flow` does, this code is more readable than its alternatives.